SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/DeviceInformationHelper.cs

```diff
@@ -40,7 +44,9 @@ internal void SetManagmentClassFactory(IManagmentClassFactory factory)
         protected override string GetSdkName()
         {
             var sdkName = WpfHelper.IsRunningOnWpf ? "appcenter.wpf" : "appcenter.winforms";
-#if NETCOREAPP3_0
+#if NET5_0            
+            sdkName = $"{sdkName}.net";
```

Comment:
```suggestion
            sdkName = $"{sdkName}.net5";
```

Reply:
Not sure about these changes because in [netcore](https://github.com/microsoft/appcenter-sdk-dotnet/pull/1549/files#diff-347334005eaa50a21480306acb2730ba9119a21a79c5547f6172e2eda9df2d3fR50) check we don't set the version in the end.